CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [0.4.9] - 2020-10-05
+### Changed
```

Comment:
```suggestion
## [0.5.0] - 2020-10-05
### Added
```
Oops, I just noticed this now - this is an addition so we should give it a minor bump. We didn't do that for a few additions earlier in the year, but I think this would be a better pattern.